src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -38,15 +46,21 @@ public static IDisposable Open(TopLevel root)
             {
                 window = new MainWindow
                 {
-                    Width = 1024,
-                    Height = 512,
+                    Width = options.Size.Width,
+                    Height = options.Size.Height,
                     Root = root,
                 };
 
                 window.Closed += DevToolsClosed;
+                window.ViewModel.ShouldVisualizeDirtyRects = options.ShouldVisualizeDirtyRects;
+                window.ViewModel.ShouldVisualizeMarginPadding = options.ShouldVisualizeMarginPadding;
+                window.ViewModel.Console.IsVisible = options.ShowConsole;
+                window.ViewModel.ShowFpsOverlay = options.ShowFpsOverlay;
+                window.ViewModel.ShowLayoutVisualizer = options.ShowLayoutVisualizer;                
```

Comment:
The side panel now also shows a style browser so the name of this needs to be changed.

Reply:
On small screens LayoutVisualizer and StyleVisualizer making the datagrid with the properties unusable because the available space is small.

Reply:
Yeah we know we need to fix the layout of DevTools. But that's a separate PR I think.